README.md

```diff
@@ -101,6 +101,14 @@ window and update the core libraries and install new packages. The extra step at
 downgrate GCC to 5.4, as the GCC6 versions in mingw currently fail to compile some of our
 dependencies. We are upgrading to a gcc-free build on Windows as soon as possible:
 
+**IMPORTANT**
+If your python is installed in other dir than `C:\Python27` you need to create a symbolic link otherwise mozjs will fail to build.
+To do it, open Command Prompt as admin and run this:
+```bat
+mkdir C:\Python27
+mklink /D C:\Python27 YOUR_PYTHON_PATH
+```
```

Comment:
I think that the right fix here is actually to do as specified in:
https://github.com/servo/servo/issues/13206#issuecomment-247177576

Which is instead to set `NATIVE_WIN32_PYTHON` to be the full path to the `python.exe` that you want to run. Thanks for looking into correcting these instructions!